microprofile/WFLY-13048_upgrade_microprofile-health_2.2.adoc

```diff
@@ -0,0 +1,64 @@
+= Upgrade MicroProfile Health to 2.2
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 3.3 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/2.2[MicroProfile Health 2.2] whose changes are documented in its https://download.eclipse.org/microprofile/microprofile-health-2.2/microprofile-health-spec.html#release_notes_2_2[release notes].
+
+This component upgrade also covers the upgrade of smallrye-health that implements MicroProfile Health 2.2.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13048[WFLY-13048]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1426[EAP7-1426]
+* Support of the `mp.health.disable-default-procedures` config property is handled separately by to https://issues.jboss.org/browse/WFLY-12952[WFLY-12952] that adds such a default server procedure for the readiness check.
+* https://github.com/wildfly/wildfly-proposals/blob/master/microprofile/WFLY-12685_upgrade_microprofile_health_2.1.0.adoc[Dev analysisy for the MicroProfile Health 2.1 upgrade]
+
```

Comment:
Asking your opinion here about adding the following one, too:
``` 
* https://github.com/wildfly/wildfly-proposals/blob/master/microprofile/WFLY-10711_health_check_extension.adoc[Dev analysis for initial integration of MicroProfile Health into Wildfly]
```
One reason for this - for instance - is that it is mentioning the location of community doc.
WDYT?

Reply:
done